pkg/handler/processor/guesser/format_guesser.go

```diff
@@ -16,12 +16,15 @@
 package guesser
 
 import (
+	"context"
+
 	"github.com/guacsec/guac/pkg/handler/processor"
-	"github.com/sirupsen/logrus"
+	"github.com/guacsec/guac/pkg/logging"
 )
 
 func init() {
-	RegisterDocumentFormatGuesser(&jsonFormatGuesser{}, "json")
+	ctx := logging.WithLogger(context.Background())
+	RegisterDocumentFormatGuesser(ctx, &jsonFormatGuesser{}, "json")
```

Comment:
I'm concerned of the use of `ctx` here because it depends on the order of `init()` which is not ideal. 

i think let's remove the `ctx` here and make `RegisterDocumentFormatGuesser` return an error with a comparable error enum, and not log errors during init? Then should be good to go.

Reply:
yea it looks like this behavior is happening and is caught by the CI https://github.com/guacsec/guac/runs/8139988072?check_suite_focus=true

Reply:
Yeah, it is because I removed the `init()` functions, so we end up with a null pointer exception...

Reply:
I put it back, now the CI is happy 😀